pkg/bwagreement/server.go

```diff
@@ -20,6 +20,7 @@ import (
 	"storj.io/storj/pkg/pb"
 	"storj.io/storj/pkg/pkcrypto"
 	"storj.io/storj/pkg/storj"
+	satellitedb "storj.io/storj/satellite/satellitedb/dbx"
```

Comment:
The comment about tally applies here as well. "satellitedb" is a specific implementation that shouldn't be used, unless you actually need to initialize it. e.g. only specific cases are 1. main.go, 2. testplanet, 3. tests.